static/js/compose.js

```diff
@@ -787,7 +787,7 @@ function validate_stream_message() {
     }
 
     var current_stream = stream_data.get_sub(stream_name);
-    var stream_count = current_stream.subscribers.num_items();
+    var stream_count = (current_stream ? current_stream.subscribers.num_items() : 0);
```

Comment:
I like the idea of moving the `@all` logic to be later in the function.  Maybe the `@all` logic could be placed in its own function, too.